lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -525,20 +507,25 @@ module.exports = class MethodMappings {
           } else {
             value = String(value);
           }
+          const element = this.getWebElement(webElement);
+          await element.sendKeys(value);
 
-          try {
-            const element = this.getWebElement(webElement);
-            await element.sendKeys(value);
+          return {
+            value: null
+          };
 
-            return {
-              value: null
-            };
-          } catch (error) {
-            return {
-              error,
-              status: -1
-            };
-          }
+        },
+
+        async uploadFile(webElementId, filepath) {
```

Comment:
can you add just this change and drop the other unrelated stuff in this file?

Reply:
cleaning up the try catch from the file, which is not required as you pointed out